docs/collisions.md

```diff
@@ -377,51 +433,15 @@ _What each unsupported character encodes to has to ultimately be decided and the
 NB: downloadFileName & downloadFolderName **should be recalculated after file or folder rename.**
 
 
-## Status
-
-
-<table>
-  <tr>
-   <td>Implementation Plan Step
-   </td>
-   <td>Status
-   </td>
-   <td> REFERENCE
-   </td>
-  </tr>
-  <tr>
-   <td>1
-   </td>
-   <td><strong>In progress</strong>
-   </td>
-   <td><a href="https://github.com/PermanentOrg/back-end/pull/11">https://github.com/PermanentOrg/back-end/pull/11</a>
-   </td>
-  </tr>
-  <tr>
-   <td>2
-   </td>
-   <td><strong>In progress</strong>
-   </td>
-   <td><a href="https://github.com/PermanentOrg/back-end/pull/11">https://github.com/PermanentOrg/back-end/pull/11</a>
-   </td>
-  </tr>
-  <tr>
-   <td>3
-   </td>
-   <td><strong>Pending 1 & 2</strong>
-   </td>
-   <td>
-   </td>
-  </tr>
-</table>
-
-
 ## Testing Plan
 
-- Test generates unique & correctly formatted `downloadName` for n colliding files in the same namespace (parent folder).
-- Test generates unique & correctly formatted `downloadName` for n colliding folders in the same namespace.
-- Test generates unique & correctly formatted `downloadName` for n colliding files & folders in the same namespace.
-- Test generates unique & correctly formatted `downloadName` for incoming files and folders holding colliding deduplication strings.
+- Test generates directory-unique & correctly formatted `downloadName` for n colliding files in the same namespace (parent folder).
```

Comment:
What's the difference between this test and the one below it?

Reply:
Reword test to make it more explicit.